.gitignore

```diff
@@ -61,3 +61,4 @@ creds.yml
 fabfile.py
 *.pyc
 .idea
+evergreen
```

Comment:
Is this just for convenience, or are we actually building a binary in the root of the repo?

Reply:
Mostly the former. It's now reasonable to type `go build main/evergreen.go` and you end up with an evergreen binary in the repo (which is reasonable, and we shouldn't commit it.)